Formula/library-manager.rb

```diff
@@ -0,0 +1,34 @@
+class LibraryManager < Formula
+  desc "Client-side web library manager"
+  homepage "https://github.com/aspnet/LibraryManager"
+  url "https://az320820.vo.msecnd.net/packages/microsoft.web.librarymanager.cli.2.0.48.nupkg"
+  sha256 "ca4f4ab4e57da62d650607b4a7a73b27a23dc2453896eda8d104ddc17e3b42a8"
+
+  resource "runtime" do
+    url "https://download.microsoft.com/download/9/1/7/917308D9-6C92-4DA5-B4B1-B4A19451E2D2/dotnet-runtime-2.1.0-osx-x64.tar.gz"
```

Comment:
This is not acceptable in core https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae. You can either make a formula out of it, or depend on the cask I guess.